std/http/cookie.ts

```diff
@@ -79,6 +84,20 @@ function toString(cookie: Cookie): string {
   return out.join("; ");
 }
 
+/**
+ * Validate Cookie property.
+ * @param key Name of the cookie.
+ * @param value Value of the cookie.
+ */
+function validateCookieProperty(
+  key: string,
+  value: string | undefined | null,
+): void {
+  if (value && !FIELD_CONTENT_REGEXP.test(value)) {
+    throw new TypeError(`The ${key} of the cookie (${value}) is invalid.`);
+  }
+}
```

Comment:
This is only good for cookie names, rename.
```suggestion
function validateCookieName(value: string | undefined | null): void {
  if (value && !FIELD_CONTENT_REGEXP.test(value)) {
    throw new TypeError(`Invalid cookie name: "${value}".`);
  }
}
```